tools/bazel-stamp-vars.sh

```diff
@@ -11,6 +11,19 @@ function onError {
   echo ""
 }
 
+# Function that throws an error if the Bazel Angular version does not match the
+# required Angular version in the project package.json file.
+function checkBazelAngularVersion {
```

Comment:
@jelbourn Not sure if we want that. I just feel like it could happen more often that we are out-of-sync with the Bazel specific variables.

I can remove that part if desired. I'm sure if we switch to Bazel completely (at some point) , there will be **only** one placeholder anyway.